### PR TITLE
[Swap] Extract `extra` content of TxModal

### PR DESCRIPTION
**Pull request by veado**
Run `yarn storybook` to check changes locally

## Preview 1

^ http://localhost:9009/?path=/story/components-modal-extra-swapassets--default

Screenshot from 2021-01-11 13-58-24

## Preview 2

^ http://localhost:9009/?path=/story/components-swap--story-default

Screenshot from 2021-01-11 13-59-11

As part of #725

### CODE REVIEW

**Review by thatStrangeGuyThorchain — CHANGES_REQUESTED** [review]

File: src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  &:last-child {
+    margin: 0;
+  }
+`
+
+export const AssetData = styled((props: AssetDataProps) => (
+  <AssetDataContainer>
+    <UIAssetData {...props} />
+  </AssetDataContainer>
+))``
```
[the remark this answers is not in the document]
Comment (veado) [reply src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx:48]: ^ Adding styles to `styled(AssetData)` does not work for any reason, that's why an extra `AssetDataContainer` is needed here.

File: src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+export const TrendContainer = styled.div`
```
[the remark this answers is not in the document]
Comment (veado) [reply src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx:50]: `className` is an artifact from Bepswap, but  it makes hard to follow and it's deprecated for same reason.

File: src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  text-align: center;
+`
+
+export const DataWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+export const StepBarContainer = styled.div`
```
[the remark this answers is not in the document]
Comment (veado) [reply src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx:26]: ^ It does not work ...

File: src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (veado) [reply src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx:48]: ^ Fixed by #730

File: src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (veado) [reply src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx:50]: ^ Fixed by #730

File: src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (veado) [reply src/renderer/components/modal/tx/extra/SwapAssets.styles.tsx:26]: ^ Fixed by #730
